### ISSUE TITLE
Shortcuts button has no outline when focused; other accessibility issues

### ISSUE THREAD

**Issue by joemaffei** [storybookjs/storybook#10862]
**Describe the bug**
When tabbing through focusable elements on the page, most UI elements seem to have some indication of focus. The Shortcuts button does not.
Other related issues: The Shortcuts panel is not scrollable using the keyboard. The search input could use a better focus indicator.

**To Reproduce**
1. Focus the browser URL
2. Hit the Tab key until the Storybook logo is focused
3. Hitting Tab one more time focuses on the button, but there's no visual indication to the user.

**Expected behavior**
The Shortcuts button should have a focus ring around it.

**Screenshots**
The Storybook logo has a focus ring:
![image](https://user-images.githubusercontent.com/9068746/82586166-c1dcf000-9b5c-11ea-9de5-ee8b03c3fe7f.png)
The Shortcuts button does not have a focus ring"
![image](https://user-images.githubusercontent.com/9068746/82586194-cc978500-9b5c-11ea-86e2-3f6ae270f822.png)
Pressing Enter does open the Shortcuts panel, however it seems impossible to scroll within it using only the keyboard.
![image](https://user-images.githubusercontent.com/9068746/82586370-0d8f9980-9b5d-11ea-9c9a-8974dd432875.png)

**System:**
System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-8705G CPU @ 3.10GHz
  Binaries:
    Node: 12.16.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.4 - C:\Program Files\nodejs\yarn.CMD
    npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.18362.449.0

(this doesn't list any of the other browsers I have installed. I found this issue while using Chrome 81)

**Comment by mkamranhamid** [comment]
Is it taken already?

**Comment by mkamranhamid** [comment]
![storybook-focus-issue](https://user-images.githubusercontent.com/11710104/82751608-ca8a2c00-9dd1-11ea-9139-a5b8a6177fa8.gif)
Is it something you're looking for? Is it the right behaviour using tabs?

**Comment by shilman** [comment]
Ooh-la-la!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.0.0-beta.22 containing PR #10944 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
